scipy-1.0/paper.tex

```diff
@@ -1078,16 +1078,16 @@ \subsection*{Governance}
 The Code of Conduct specifies how breaches can be reported to a Code of Conduct Committee, and outlines procedures for the Committee's response. Our Diversity Statement ``welcomes and encourages participation by everyone''.
 
 
-\subsection*{Community beyond the SciPy library}
+\subsection*{Relation with downstream projects}
```

Comment:
How about just "Downstream Projects"? 

The section is called "Project organization and community", and I think what we're trying to say with these subsection headings is that the project is organized into three groups: 

- "Governance", in the sense of "the persons/committees /departments who make up a body for the purpose of administering something",
- "Maintainers and Contributors", and
- "Downstream Projects".

"Relation with downstream projects" is not a group (of people) like the first two are.

Reply:
Done